packages/compass-e2e-tests/helpers/commands/connect.ts

```diff
@@ -188,9 +188,10 @@ export async function waitForConnectionResult(
     }
   } else if (connectionStatus === 'failure') {
     if (TEST_MULTIPLE_CONNECTIONS) {
-      const element = await browser.$(Selectors.ConnectionToastErrorText);
-      await element.waitForDisplayed(waitOptions);
-      return await element.getText();
+      await browser
+        .$(Selectors.ConnectionToastErrorText)
+        .waitForDisplayed(waitOptions);
+      return await browser.$(Selectors.LGToastTitle).getText();
```

Comment:
This was accidentally returning the toast body text (which is always the same) and not the toast title text. Nothing cared about it yet.